### PR TITLE
Update LUIS Runtime swagger to an auto generated version

### PR DESCRIPTION
**Pull request by moabba**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by moabba** [comment]
@yangyuan Do you know if there's a reason that we don't have java and .Net in the `swagger-to-sdk` section? Should I add them? Apparently Java wasn't updated in a long time.

File: specification/cognitiveservices/data-plane/LUIS/Runtime/preview/v3.0/LUIS-Runtime.json [hunk 1]
```diff
@@ -76,43 +98,70 @@
       "post": {
         "description": "Gets the predictions for an application version.",
         "operationId": "Prediction_GetVersionPrediction",
+        "consumes": [
```
Comment (MyronFanQiu) [inline specification/cognitiveservices/data-plane/LUIS/Runtime/preview/v3.0/LUIS-Runtime.json]: @sergey-shandar I want to confirm this line. It add a new field `consumes` into the method which seem like bring a constraint. Is this a breaking change or not?

**Comment by MyronFanQiu** [comment]
LGTM. Most of commits are changes of the order of the code.

File: specification/cognitiveservices/data-plane/LUIS/Runtime/preview/v3.0/LUIS-Runtime.json (hunk 1 above)
Comment (moabba) [reply specification/cognitiveservices/data-plane/LUIS/Runtime/preview/v3.0/LUIS-Runtime.json]: According to the "BreakingChange" check it's not. This was missing from the original swagger file and nothing new was actually introduced to the service. We've been supporting `applications/json` only from day one. 

I tried to generate the C# SDK from the new file and checked the JS and Go SDKs and none of them had any breaking changes.

File: specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md [hunk 2]
```diff
@@ -13,7 +13,6 @@ go:
 
 ``` yaml $(go) && $(multiapi)
 batch:
-  - tag: runtime_2_0
```
Comment (MyronFanQiu) [inline specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md]: @jhendrixMSFT Please review this change.

**Comment by MyronFanQiu** [comment]
@yangyuan Please review this PR. Thanks

File: specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md (hunk 2 above)
Comment (jhendrixMSFT) [reply specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md]: Why is this removed?

Comment (moabba) [reply specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md]: This results in a conflict while generating the SDK with autorest. Here's a result of the failed build https://dev.azure.com/azure-sdk/public/_build/results?buildId=51155

We also generate only the runtime_3_0_preview in the other SDKs and only do the batch in Go. So this actually results in an inconsistent SDK release cycle.

Comment (jhendrixMSFT) [reply specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md]: We support generating multiple API versions in various languages.  Are you saying this API version is no longer supported?
The generation failure appears to be a regression in autorest, I'm following up on that.

Comment (moabba) [reply specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md]: No, it's still supported. But the default tag for other languages is the `runtime_3_0_preview`. Anyway, if the autorest regression is fixed, it's okay to revert this.

Comment (jhendrixMSFT) [reply specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md]: @NelsonDaniel this appears to be a regression in autorest, can you please take a look?

Comment (MyronFanQiu) [reply specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md]: @NelsonDaniel Hi. Any update about this PR ?

Comment (NelsonDaniel) [reply specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md]: Fixed!

Comment (NelsonDaniel) [reply specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md]: @MyronFanQiu You can add the tag back

Comment (jhendrixMSFT) [reply specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md]: I rebased on top of master and added the tag back.

**Review by mmyyrroonn — APPROVED** [review]

**Comment by MyronFanQiu** [comment]
@yangyuan @DavidLiCIG Hi. Please review these changes when you are free. If you are ok with this PR, I plan to merge it this week.

**Comment by yangyuan** [comment]
By reading the Go SDK PR, this PR doesn't contain any actual change. Basically just reformat. Can you confirm?

**Comment by moabba** [comment]
> By reading the Go SDK PR, this PR doesn't contain any actual change. Basically just reformat. Can you confirm?

@yangyuan Yes exactly. Before, we wrote this swagger file manually by hand. We made changes in our pipeline to auto-generate this from our code. So the generation shuffled some things around. But this is basically to make sure that any actual changes would be clear in the future and won't include this shuffling.

**Review by yangyuan — APPROVED** [review]

**Pull request merged**